specs/tilebag_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'spec_helper'
+
+describe "Scrabble" do
+
+  describe "TileBag" do
+
+    describe "intiialize" do
+      it "initializes tilebag as an instance of TileBag class" do
+        Scrabble::TileBag.new.must_be_instance_of Scrabble::TileBag
+      end
+
+      it "initializes tilebag instance with default tiles" do
+        Scrabble::TileBag.new.default_tiles.must_include 'a'
+        Scrabble::TileBag.new.default_tiles.must_include 'o'
+        Scrabble::TileBag.new.default_tiles.keys.must_include 'u'
+      end
+    end
+
+    describe "draw_tiles" do
+      it "returns num random tiles" do
+        Scrabble::TileBag.new.draw_tiles(1).length.must_equal 1
+      end
+
+      it "must return an array of tiles" do
+        Scrabble::TileBag.new.draw_tiles(1).must_be_kind_of String
+      end
```

Comment:
This test doesn't match its name! The name claims it's looking for an array, but you check for a string.

Reply:
You're missing a couple of interesting edge cases here:
- What happens if you attempt to draw 0 tiles?
- Are you able to draw down to 0 tiles remaining?
- What happens when you attempt to draw more tiles than are in the bag?